src/sentry/models/authidentity.py

```diff
@@ -19,6 +19,8 @@ class AuthIdentity(Model):
     last_synced = models.DateTimeField(default=timezone.now)
     date_added = models.DateTimeField(default=timezone.now)
 
+    identity = FlexibleForeignKey('sentry.Identity', null=True)
```

Comment:
ill probably back this out for now, as theres no reason to change this table until we're ready.